pkg/utils/utils.go

```diff
@@ -1,6 +1,7 @@
 package utils
 
 import (
+	// #nosec G501
```

Comment:
What does this mean?

Reply:
Disables rule #G501 from: https://github.com/securego/gosec#available-rules